lib/datasource/remote/model/member_model.dart

```diff
@@ -1,29 +1,32 @@
-import 'package:equatable/equatable.dart';
+import 'package:flutter/foundation.dart';
 import 'package:seeds/datasource/remote/model/serialization_helpers.dart';
 import 'package:seeds/domain-shared/user_citizenship_status.dart';
 
-class MemberModel extends Equatable {
+class MemberModel {
   final String account;
   final String nickname;
   final String image;
+  final String status;
   final bool isSeedsUser;
-  final UserCitizenshipStatus? citizenshipStatus;
 
   const MemberModel({
     required this.account,
     required this.nickname,
     required this.image,
+    required this.status,
     this.isSeedsUser = true,
-    this.citizenshipStatus,
   });
 
+  UserCitizenshipStatus get userCitizenshipStatus {
+    return enumFromString<UserCitizenshipStatus>(UserCitizenshipStatus.values, status);
+  }
+
   factory MemberModel.fromJson(Map<String, dynamic> json) {
     return MemberModel(
       account: json['account'],
       nickname: json['nickname'],
       image: json['image'],
-      citizenshipStatus: enumFromString<UserCitizenshipStatus>(
-          UserCitizenshipStatus.values, hasEmptyValue<String>('status', json)),
+      status: json['status'] ?? '',
     );
```

Comment:
@n13  The `status` field in the member model always returns a default value? I mean it will never be null?

Reply:
Yeah the chain calls never return null, they return "" if the field is not set